### PR TITLE
completed fetch exercise

### PR DESCRIPTION
**Pull request by Ajowyang**
Screenshot 2024-08-08 at 11 26 09 AM

### CODE REVIEW

File: fetch/main.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+async function fetchData(): Promise<void> {
+  try {
+    const response = await fetch('https://jsonplaceholder.typicode.com/users');
+
+    if (!response.ok) {
+      throw new Error(`HTTP error! Status: ${response.status}`);
+    }
+
+    const data = (await response.json()) as Pokemon;
```
Comment (brandon-moy) [inline fetch/main.ts]: You have a Pokemon interface here but you are sending a request to the `jsonplaceholder.typicode.com/users`. These users aren't Pokemon. I would either say that you should change your interface to `User` and update the properties with users, or update your `fetch` url to send a request to the `pokeAPI`

**Review by brandon-moy — CHANGES_REQUESTED** [review]
Quiz questions! And one change with your `interface` or `fetch` url

**Comment by Ajowyang** [comment]
Screenshot 2024-08-08 at 12 15 44 PM

**Review by brandon-moy — APPROVED** [review]
👍

**Pull request merged**